CHANGES

```diff
@@ -1,5 +1,11 @@
 Unreleased
 ==========
+- Update for Wagtail >5.1
+- Drop support for Django <4.2
```

Comment:
Technically Django 3.2 has security support until April, but by the time we get to doing a release that probably won't be far off.